src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
Although true, there is no reason at all for an external endpoint to call this method, as canLogin already covers it. Exposing another endpoint which arbitrarily returns the same, is adding unnecessary options. `canLogin` covers it all, so with that, there's no need to expose another API that does part of that separately. Moving from an arbitrary "isLockedOut" to a more logical, all encapsulated "canLogin" is the most logical way forward and with a protected method, can be overridden if needed.

This reduces the API while giving the same amount of flexibility.